src/lib.rs

```diff
@@ -444,6 +444,10 @@ impl Builder {
             output_vector.push("--no-prepend-enum-name".into());
         }
 
+        if self.options.array_pointers {
```

Comment:
This should be probably more argument-specific (so `--use-array-pointers-in-arguments` / `array_pointers_in_arguments`)?

Reply:
Done